set.mm

```diff
@@ -209475,6 +209476,37 @@ is always a subcategory (and it is full, meaning that all morphisms of
       QWJBFVRWKWLWNWOVCZVDVQXBXCXDUTVEVQVSBVFPNWAVNULXFEBVSVGUJVHAVPVIVJVKVL $.
   $}
 
+  ${
+    setc2ohom.c $e |- C = ( SetCat ` 2o ) $.
+    ${
+      setc2obas.b $e |- B = ( Base ` C ) $.
+      $( ` (/) ` and ` 1o ` are distinct objects in ` ( SetCat `` 2o ) ` .
+         This combined with ~ setc2ohom demonstrates that the category does not
+         have pairwise disjoint hom-sets.  See also ~ df-cat and ~ cat1 .
+         (Contributed by Zhi Wang, 24-Sep-2024.) $)
+      setc2obas $p |- ( (/) e. B /\ 1o e. B /\ 1o =/= (/) ) $=
+        ( c0 wcel c1o wne cpr 0ex prid1 cbs cfv c2o wceq wtru cvv 2oex eleqtrri
+        a1i setcbas mptru df2o3 3eqtr2i 1oex prid2 1n0 3pm3.2i ) EAFGAFGEHEEGIZ
+        AEGJKABLMZNUIDNUJOPBNQCNQFPRTUAUBUCUDZSGUIAEGUEUFUKSUGUH $.
+    $}
+
+    setc2ohom.h $e |- H = ( Hom ` C ) $.
+    $( ` ( SetCat `` 2o ) ` is a category (provable from ~ setccat and ~ 2oex )
+       that does not have pairwise disjoint hom-sets, proved by this theorem
+       combined with ~ setc2obas .  Notably, the empty set ` (/) ` is
+       simultaneously an object ( ~ setc2obas ) , an identity morphism from
+       ` (/) ` to ` (/) ` , and a non-identity morphism in from ` (/) ` to
```

Comment:
"in"?

Reply:
https://github.com/metamath/set.mm/pull/4239/commits/42b22d4d4b20168e2b342015c0cee7c685ab2524